tests/unit/instance-initializers/embedded-test.ts

```diff
@@ -49,7 +49,8 @@ module('Unit | Instance Initializer | embedded', function (hooks) {
       this.appInstance.resolveRegistration('config:environment'),
       {
         APP: {},
-      }
+      },
+      /* block Prettier */
```

Comment:
Not sure to understand why it is needed

Reply:
Otherwise it renders

```ts
    assert.deepEqual(this.appInstance.resolveRegistration('config:environment'), {
      APP: {},
    })
```

which is garbage and less easy/smooth to read.

Reply:
Not sure we should keep it, maybe sticking with what is formatted by prettier for all files is better 🤷 but ok for me if you want to keep this comment to have the code formatted in a beautiful way!